### PR TITLE
Update language in UG

### PR DESCRIPTION
**Pull request by Zacchua**
Update language use for picture command to exit command

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -515,72 +515,72 @@ Returns all contacts who have a Telegram handle pegged with them.
...
 
-Adds a profile picture to an existing contact or changes the existing profile picture if available.
+This command adds a profile picture to an existing contact or changes the existing profile picture if available.
 
 <div markdown="block" class="alert alert-primary">
 **:mag_right: Format:**
 `picture INDEX`
-* Opens file browser allowing you to choose any .jpg or .png file as the profile picture.
-* Adds the selected picture to the contact at the specified `INDEX`.
+* A file browser will be opened allowing you to choose any .jpg or .png file as the profile picture.
```
Comment (stanley-1) [inline docs/UserGuide.md:523]: Add a comma after "opened"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -515,72 +515,72 @@ Returns all contacts who have a Telegram handle pegged with them.
...
+* The selected picture will be added to the contact at the specified `INDEX`.
```
Comment (stanley-1) [inline docs/UserGuide.md:524]: Maybe we can change this to "Your selected picture..."

File: docs/UserGuide.md [hunk 3]
```diff
@@ -664,7 +664,7 @@ Only the remark “Hates carrots” is added to Annie Wan’s listing in the app
 
 ### View help: `help` <a name="help"></a>
 
-Shows a message explaining how to access the User Guide or obtain in-app guidance for selected commands.
+This command shows a message explaining how to access the User Guide or obtain in-app guidance for selected commands.
```
Comment (stanley-1) [inline docs/UserGuide.md:667]: Can consider changing to: "This command displays a message that shows you how you can access our User Guide or obtain in-app guidance for selected commands."

File: docs/UserGuide.md [hunk 4]
```diff
@@ -676,7 +676,7 @@ Shows a message explaining how to access the User Guide or obtain in-app guidanc
 
 ### View in-app guidance: `help COMMAND` <a name="help_command"></a>
 
-Displays concise instructions for users to learn the given `COMMAND`'s function.
+This command displays concise instructions for users to learn the given `COMMAND`'s function.
```
Comment (stanley-1) [inline docs/UserGuide.md:679]: Change to: "This command displays concise instructions within the app for you to learn the given `COMMAND`'s function."

**Review by stanley-1 — APPROVED** [review]
Thanks Zac! Just a couple of areas to amend as listed below!

**Comment by Zacchua** [comment]
Thanks for the feedback @stanley-1 ! I have made the necessary changes :-)

**Comment by david-eom** [comment]
Did a quick run through, LGTM, will be merging now!

**Pull request merged**